wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1538,7 +1540,8 @@ class TryExceptMultipleReturnPathViolation(ASTViolation):
 @final
 class ComplexDefaultValuesViolation(ASTViolation):
     """
-    Forbids to use anything that is not a ``ast.Name`` or ``ast.Attribute`` as
+    Forbids to use anything that is not a `ast.Name`, `ast.Attribute`,
```

Comment:
We use double ` chars to indicate that it is a code.
That's how sphinx renders it.